src/utils/polygon.cpp

```diff
@@ -355,6 +355,11 @@ void PolygonRef::simplify(const coord_t smallest_line_segment_squared, const coo
         accumulated_area_removed += removed_area_next;
 
         const coord_t length2 = vSize2(current - previous);
+        if (length2 < 25)
```

Comment:
I initially thought that this would cause the long lines to become shifted again. To my suprise, this doesn't happen...

Reply:
No, because previously this was happening _only_ for the long lines.